packages/fxa-auth-server/lib/senders/emails/templates/subscriptionCancellation/en.ftl

```diff
@@ -7,5 +7,6 @@ subscriptionCancellation-title = Sorry to see you go
 #   $invoiceTotal (String) - The amount of the subscription invoice, including currency, e.g. $10.00
```

Comment:
You need to change this comments into a group-level comment, since you're splitting up the string in multiple strings (two hashes, leave an empty line before and after the group of comments).

Reply:
I've updated this as well. Thank you!